frontend/taipy-gui/src/components/Taipy/Metric.tsx

```diff
@@ -21,111 +21,123 @@ import Skeleton from "@mui/material/Skeleton";
+    const metricDelta = useDynamicProperty(props.delta, props.defaultDelta, undefined)
+    const className = useClassNames(props.libClassName, props.dynamicClassName, props.className);
+    const baseLayout = useDynamicJsonProperty(props.layout, props.defaultLayout || "", emptyLayout);
+    const baseStyle = useDynamicJsonProperty(props.style, props.defaultStyle || "", defaultStyle);
 
-  const deltaValueSuffix = useMemo(() => {
-    switch (props.formatDelta) {
-      case "%.2f%%":
-        return "%";
-      default:
-        return "";
-    }
-  }, [props.formatDelta]);
+    const deltaValueSuffix = useMemo(() => {
+        switch (props.formatDelta) {
+            case "%.2f%%":
+                return "%";
+            default:
+                return "";
+        }
+    }, [props.formatDelta]);
 
-  const formatType = useMemo(() => {
-    switch (props.format) {
-      case "%.2f%%":
-        return "%";
-      default:
-        return "";
-    }
-  }, [props.format]);
+    const formatType = useMemo(() => {
+        switch (props.format) {
+            case "%.2f%%":
+                return "%";
+            default:
+                return "";
+        }
+    }, [props.format]);
 
-  const metricType = useMemo(() => {
-    switch (props.type) {
-      case "circular":
-        return "angular";
-      case "linear":
-        return "bullet";
-      default:
-        return "angular";
-    }
-  }, [props.type]);
+    const metricType = useMemo(() => {
+        switch (props.type) {
+            case "circular":
+                return "angular";
+            case "linear":
+                return "bullet";
+            default:
+                return "angular";
+        }
+    }, [props.type]);
 
-  const extendedDelta = useMemo(() => {
-    const refValue = typeof metricValue === 'number' && typeof metricDelta === 'number' ? metricValue - metricDelta : undefined;
-    return {
-      reference: refValue,
-      suffix: deltaValueSuffix,
-    };
-  }, [metricValue, metricDelta, deltaValueSuffix]);
+    const extendedDelta = useMemo(() => {
+        const refValue = typeof metricValue === 'number' && typeof metricDelta === 'number' ? metricValue - metricDelta : undefined;
+        return {
+            reference: refValue,
+            suffix: deltaValueSuffix,
+        };
+    }, [metricValue, metricDelta, deltaValueSuffix]);
 
-  const data = useMemo(() => ([
-    {
-      domain: {x: [0, 1], y: [0, 1]},
-      value: metricValue,
-      type: "indicator",
-      mode: "gauge+number+delta",
-      number: {
-        suffix: formatType
-      },
-      delta: extendedDelta,
-      gauge: {
-        axis: {range: [props.min, props.max]},
-        shape: metricType,
-        threshold: {
-          line: {color: "red", width: 4},
-          thickness: 0.75,
-          value: metricThresholdValue
+    const data = useMemo(() => ([
+        {
+            domain: {x: [0, 1], y: [0, 1]},
```

Comment:
This doesn't have to always be [0, 1]. We can adjust these values to position your plot anywhere within the plotting area. But since we do not have a clear requirement on this yet so I would leave the plot spans the entire width and height of the plotting area